src/Http/Rememberer.php

```diff
@@ -49,9 +47,7 @@ public function rememberUser(ResponseInterface $response, $userId)
 
     public function forget(ResponseInterface $response)
     {
-        return FigResponseCookies::set(
-            $response,
-            $this->cookie->expire(self::COOKIE_NAME)
-        );
+        return $this->cookie->expire(self::COOKIE_NAME)
```

Comment:
As per the above concerns with nullable value, I'd like to see a unit test for this method